nix_update/dependency_hashes.py

```diff
@@ -35,14 +35,15 @@ def extract_hash_from_nix_error(stderr: str) -> str | None:
     Handles various formats:
     - got:    xxx
     - got:    sha256-xxxxx=
+    - got:    sha256:xxxxx
```

Comment:
```suggestion
    - got:    sha256:xxxxx
    - got:    sha256-xxxxx=
```
lets keep the sri and none sri hashes separate